crates/turborepo-lib/Cargo.toml

```diff
@@ -60,6 +60,7 @@ jsonc-parser = { version = "0.21.0" }
 lazy_static = { workspace = true }
 libc = "0.2.140"
 notify = "5.1"
+path-clean = "1.0.1"
```

Comment:
This library was intentionally written to match the behavior of Go's `filepath.Clean`. I did a spot check and it does seem to be a true to form port, but more eyes are always welcome.
[filepath.Clean](https://cs.opensource.google/go/go/+/refs/tags/go1.21.4:src/path/filepath/path.go;l=98)
[path_clean::clean](https://github.com/danreeves/path-clean/blob/master/src/lib.rs#L58)